content/faq/how-rotate-kubernetes-ssh-keys.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "How to rotate ssh keys for Kubernetes?"
+description: "Learn how to rotate ssh keys for kubernetes."
+tags:
+- kops
+- geodesic
+- kubernetes
+- faq
+---
+
+# Question
+
+How to rotate ssh keys for Kubernetes?
+
+# Answer
+
+Geodesic creates ssh keys with [terraform-aws-key-pair]({{< relref "terraform-modules/security/terraform-aws-key-pair.md" >}}) module.
```

Comment:
Better to say:
We provision SSH keys in `geodesic` using [terraform-aws-key-pair]({{< relref "terraform-modules/security/terraform-aws-key-pair.md" >}}) module